src/java/main/org/apache/zookeeper/server/persistence/Util.java

```diff
@@ -294,5 +297,25 @@ public int compare(File o1, File o2) {
         Collections.sort(filelist, new DataDirFileComparator(prefix, ascending));
         return filelist;
     }
+
+    /**
+     * Returns true if file is a log file.
+     *
+     * @param file
+     * @return
+     */
+    public static boolean isLogFile(File file) {
+        return file.getName().startsWith(LOG_FILE_PREFIX);
```

Comment:
It might be safer to check here with the dot (".") included in the prefix.

Reply:
I agree that it might be safer to check with the dot. I'll add this change.

Reply:
This is done.